skietbaan_fe/src/components/CreatePage.js

```diff
@@ -22,55 +23,55 @@ class CreatePage extends Component {
 	render() {
 		return (
 			<div>
-				<div>
+				<Container id="create-container">
+
 					<Collapsible
 						id="Competitions"
 						trigger={
 							<a href="">
-								<Button variant="outline-secondary" size="lg" width="125px">
+								<Button variant="outline-secondary" size="md" width="125px">
 									<strong>Competitions</strong>
 								</Button>
 							</a>
 						}
 					>
+						<Container id="competition-container" />
 						<Tab.Container>
-							<Nav className="justify-content-center" fill variant="tabs" defaultActiveKey="/home">
+							<Nav className="justify-content-center" fill variant="tabs" defaultActiveKey="">
 								<Nav.Item>
 									<Nav.Link eventKey="first">COMPETITION</Nav.Link>
 								</Nav.Item>
 								<Nav.Item>
 									<Nav.Link eventKey="second">VIEW COMPETITION</Nav.Link>
 								</Nav.Item>
 							</Nav>
-
-							<Tab.Content>
-								<Tab.Pane eventKey="first">
-									<p>
-										Against that time, if ever that time come, When I shall see thee frown on my
-										defects, When as thy love hath cast his utmost sum, Call'd to that audit by
-										advis'd respects; Against that time when thou shalt strangely pass, And scarcely
-										greet me with that sun, thine eye, When love, converted from the thing it was,
-										Shall reasons find of settled gravity; Against that time do I ensconce me here,
-										Within the knowledge of mine own desert,
-									</p>
-								</Tab.Pane>
-							</Tab.Content>
-							<Tab.Content>
-								<Tab.Pane eventKey="second">
-									<p>
-										I never saw that you did painting need, And therefore to your fair no painting
-										set; I found, or thought I found, you did exceed That barren tender of a poet's
-										debt: And therefore have I slept in your report, That you yourself, being
-										extant, well might show How far a modern quill doth come too short, Speaking of
-										worth, what worth in you doth grow. This silence for my sin you did impute,
-										Which shall be most my glory being dumb;
-									</p>
-								</Tab.Pane>
-							</Tab.Content>
+							<Container>
+								<Row>
+									<Col>
+										<Tab.Content>
+											<Tab.Pane eventKey="first">
+												<p>
+													<CreateComp />
```

Comment:
What happened to the content?

Reply:
Content is from the <CreateComp/> (CreateComp.js), and so will the contents for other navs

Reply:
Okay they were just not in the PR so that is why I was unsure